eng/pipelines/common/templates/jobs/validate-signed-package-job.yml

```diff
@@ -106,6 +106,35 @@ jobs:
       }
     displayName: 'Verify nuget signature'
 
+  - powershell: |
```

Comment:
Shouldn't this be skipped on other runs except the official pipeline with the official template?

Reply:
added